### ISSUE TITLE
No _resource.lua file in new version

### ISSUE THREAD

**Issue by FPFrosty** [TomGrobbe/vMenu#151]
First off, I have to say that I love this menu!  Great job!

I have only been running v2.2.1 for a few weeks now.  As soon as I saw that there was a new release, I downloaded it and installed on my server.  Since I was upgrading, I copied the new files into the directory as instructed.  I left my config files intact from v2.2.1.  Anxious to try now the new features, I jumped into my server only to realize I couln't access any new/changed features.  After looking through the files, I noticed that there wasn't a _resource.lua file in the new download.  Is this something that was overlooked, or am I supposed to make my own?  I was going to see if I could modify the v2.2.1 _resource.lua file but didn't want to screw it up.  I have just recently started playing with installing mods, but have never tried to create any myself.  Any help would be greatly appreciated!

Thanks!

**Comment by TomGrobbe** [comment]
@FPFrosty The v2.2.2 download does contain a __resource.lua, are you sure you didn’t download the source files by accident?

**Comment by FPFrosty** [comment]
I'm not sure how, but I did have the source files.  I always watch the file names to ensure that I don't do that, but I seemed to anyway.  What's sad, is that I thought they could be source files due to the plethora of additional files.  So, I downloaded it again.  Same results.  I must be blind.

I appreciate the help and the fast response.  I hate bugging devs for stupid questions like this.  I played with it for a couple of days before reaching out.

Thanks again for the help and the awesome menu!
